core/serialize/serialize.cc

```diff
@@ -831,51 +837,6 @@ TypeParameter SerializerImpl::unpickleTypeParameter(UnPickler &p, const GlobalSt
     return result;
 }
 
-Pickler SerializerImpl::pickle(const GlobalState &gs) {
-    Timer timeit(gs.tracer(), "pickleGlobalState");
-    Pickler result;
-    result.putU4(Serializer::VERSION);
-    result.putU4(gs.kvstoreUuid);
```

Comment:
There's no longer a path where we check the UUID when deserializing a whole `GlobalState`, but that's fine as we were only using this function for the payload at the point of this change. We do validate the UUID when deserializing the name table from the KeyValueStore, and nothing has changed with that path.